tests/e2e/pages/rancher-extensions.page.ts

```diff
@@ -19,11 +19,17 @@ export class RancherExtensionsPage extends BasePage {
 
     // Enable extensions
     await this.page.getByRole('button', { name: 'Enable' }).click();
-    await this.ui.checkbox('Rancher Extensions Repository').setChecked(rancherRepo)
-    await this.ui.checkbox('Partners Extensions Repository').setChecked(partnersRepo)
-    await this.page.getByRole('button', { name: 'OK' }).click();
+    await expect(this.page.getByRole('heading', { name: 'Enable Extension Support?' })).toBeVisible()
+
+    // Add repositories
+    await this.ui.checkbox('Rancher Extension').setChecked(rancherRepo)
```

Comment:
I can't find any reference to this label `Rancher Extension` for the repository, perhaps just targeting the `data-testid` for [that checkbox](https://github.com/rancher/dashboard/blob/b9180782e23fb67edb7d53dc9769c60573799434/shell/pages/c/_cluster/uiplugins/SetupUIPlugins.vue#L296) would work?

![ext1](https://github.com/rancher/kubewarden-ui/assets/40806497/97aff1c6-974d-4d75-a677-0b06f337c4fe)

Reply:
This is selecting checkboxes based on partial text next to the checkbox:
 - Add Official `Rancher Extension` Repository
 - Add `Partners Extensions` Repository

2.7.7 renamed `Add Rancher Extension Repository` -> `Add Official Rancher Extensions Repository`, this text matches them both.

There are no data-testids on 2.7.6, probably added in 2.7.7.
If it's confusing I can create separate locators for 2.7.6 and 2.7.7.

Reply:
Ah I see, I assumed it was matching the text exactly. I think this will suffice, I doubt the wording will change.